create-user.py

```diff
@@ -0,0 +1,6 @@
+from app import app
+from flask import Flask, render_template
+
+@app.route('/api/auth/register', methods = ['POST'])
+def create_account():
+    return render_template('create-user.html')
```

Comment:
In future, remember that all files end with an empty line

Reply:
I'll consider that. Thanks.